specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/execution.json

```diff
@@ -0,0 +1,1103 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-05-01",
+    "title": "Execution Service API"
+  },
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/execution/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/experiments/{experimentName}/localrun": {
+      "post": {
+        "tags": [
+          "Execution"
+        ],
+        "summary": "Start a run on a local machine.",
+        "description": "Starts an experiment run using the provided definition.json file to define the run.\r\n            The source code and configuration is defined in a zip archive in project.zip.",
+        "operationId": "Execution_StartLocalRun",
+        "consumes": [
```

Comment:
We don't need to do the multipart upload pattern for Execution_StartLocalRun. It doesn't need the zip archive, just the JSON body.

While we can't change the existing API, we can skip documenting it in favor of a new API for starting a local run that takes a JSON definition structure directly in the body.

Reply:
done